Makefile

```diff
@@ -18,11 +18,5 @@ build:
 test:
 	xcodebuild test $(XC_ARGS) | $(XCPRETTY)
 
-xcbuild:
-	xctool $(XC_ARGS)
```

Comment:
Removed because xctool doesn’t support building for Xcode 8 and later. Facebook recommends updating to xcodebuild or other build tools. 
https://github.com/facebook/xctool/issues/704